dropshot/src/lib.rs

```diff
@@ -187,13 +187,14 @@
  *      [query_params: Query<Q>,]
  *      [path_params: Path<P>,]
  *      [body_param: TypedBody<J>,]
+ *      [body_param: UntypedBody<J>,]
```

Comment:
nit:
```suggestion
 *      [body_param: UntypedBody,]
```

Reply:
Nice catch.  Fixed in eccf949da8c52d3d56278bb1ba8478d102695e82.